hibike/hibikeDevices.json

```diff
@@ -92,7 +92,7 @@
                                                           {"number": 7  , "name": "natsuki"    , "type": "uint64_t", "read": true , "write": true  },
                                                           {"number": 8  , "name": "yuko"       , "type": "int64_t" , "read": true , "write": true  },
                                                           {"number": 9  , "name": "mizore"     , "type": "float"   , "read": true , "write": true  },
-                                                          {"number": 10 , "name": "nozomi"     , "type": "double"  , "read": true , "write": true  },
+                                                          {"number": 10 , "name": "nozomi"     , "type": "float"  , "read": true , "write": true  },
```

Comment:
Nope.  If you write "float" here, python will be expecting 4 bytes.  Which is the same as arduino's double.